src/FFMpeg/Filters/Video/ResizeFilter.php

```diff
@@ -98,8 +98,10 @@ public function apply(Video $video, VideoInterface $format)
         if (null !== $dimensions) {
             $dimensions = $this->getComputedDimensions($dimensions, $format->getModulus());
 
-            $commands[] = '-s';
-            $commands[] = $dimensions->getWidth() . 'x' . $dimensions->getHeight();
+            // Using Filter to have ordering
+            $commands[] = '-vf';
+            $commands[] = '[in]scale=' . $dimensions->getWidth() . ':' . $dimensions->getHeight() . ' [out]';
```

Comment:
because it is used to be merged outside . it is the best practice to have in and out in filter command .
but if you have only one filter . you can just ignore the in and out. because it must be in in front and out behind. but we need to do the orderinga and multiple filters. it s better to specify the in and out.